test/CMakeLists.txt

```diff
@@ -151,15 +153,18 @@ target_include_directories(TestScan
 blt_add_test( NAME TestScan
               COMMAND TestScan )
 
-blt_add_executable( NAME Benchmarks
-                    SOURCES Benchmarks.cpp
-                    DEPENDS_ON ${care_test_dependencies} )
+if (CARE_ENABLE_LOOP_FUSER)
```

Comment:
Just move this test up by the other ones so at a glance it is obvious that it exercises the same functionality.

Reply:
That makes sense, changed

Reply:
Thanks!